packages/web/src/index.tsx

```diff
@@ -13,4 +12,8 @@ import './index.css'
 // when running in dev mode.
 import Root from './root'
 
-ReactDOM.render(<Root />, document.getElementById('root'))
+const container = document.getElementById('root')
+if (container) {
```

Comment:
Is falsy container a realistic case, or just to help type checking?
I assume if we skip the root render, the site doesn't load at all

Reply:
for us it's impossible unless we update the index.html file and remove the "root" element, was just a type thing